src/devices/Arduino/README.md

```diff
@@ -0,0 +1,47 @@
+# SPI, GPIO and I2C drivers for Arduino
+
+This binding supports GPIO and I2C access from a normal Desktop environment (Windows, Linux) trough an Arduino board.
+
+## Device family
+
+This binding remotely controls different Arduino boards directly from PC Software. It provides support for accessing GPIO ports as well as I2C devices. The Arduino is remote controlled by individual commands from the PC, the entire program will run on the PC, and not on the Arduino, so the connection cannot be removed while the device is being used. 
+
+## Desktop Requirements
+
+In order to have an Arduino board working with the PC, you need to Install the Arduino IDE together with the drivers for your board type. If you get a simple sketch uploaded and running (such as the blinking LED example) you are fine to start. 
+
+## Preparing your Arduino
+You need to upload a special sketch to the Arduino. This sketch implements the "Firmata-Protocol", a communication protocol that allows to remotely control all the inputs and outputs of the board. See https://github.com/firmata/protocol/blob/master/protocol.md for details. 
+
+The binding requires Firmata Version 2.6, which is implemented i.e. by the ConfigurableFirmata project. 
```

Comment:
OK, so I understand now why the .ino is part of the project. Still what is specific to the .ino that is not part of the main Firmata project?

Reply:
ConfigurableFirmata uses another top-level architecture than firmata. It uses an OO abstraction layer arount the firmata library that allows to selectively enable or disable features. See http://firmatabuilder.com/. 
Also, the code is much better structured and readable, which helps in understanding the command handling on the Arduino side if something doesn't work as expected.